operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -352,8 +368,11 @@ Or
 cray ims images update bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb --metadata-operation remove --metadata-key sbps-project
 ```
 
-**Note:** As mentioned in `Image tagging` step, `rootfs` are tagged automatically by BOS, for untagging an image, we need to use above `craycli`
-utility of IMS as untagging of an image is not supported by BOS.
+**Note #1:** Only run the `untag` command on images that are not currently in use. Untagging images that are currently in use will
+stop the content projection by SBPS Marshal agent causing undesirable behavior on compute/UAN nodes using the content.
+
+**Note #2:** As mentioned in the `Image tagging` step, `rootfs` is tagged automatically by BOS, for untagging an image,
+we need to use above `craycli` utility of IMS as untagging of an image is not supported by BOS.
```

Comment:
```suggestion
use the aforementioned `craycli` utility of IMS because BOS does not support automatically untagging an image.
```

Reply:
Just a query: Does BOS support any manual way of untagging an image ?

Reply:
No, the manual method is via `craycli`